pkg/minikube/bootstrapper/bsutil/ktmpl/v1beta1.go

```diff
@@ -36,8 +36,10 @@ bootstrapTokens:
 nodeRegistration:
   criSocket: {{if .CRISocket}}{{.CRISocket}}{{else}}/var/run/dockershim.sock{{end}}
   name: "{{.NodeName}}"
+{{- if .NodeIP }}
```

Comment:
I'm wondering about rendering these things like NodeIP with ipv6 stuff in them, and `:` which breaks yaml? For example `[::]:10249` ?

I see you have quotes elsewhere, but maybe a few spots are missing?